communication/contributor-comms/blogging-resources/README.md

```diff
@@ -1,5 +1,7 @@
 # Blogging resources
 
+Contributor Comms depends on great storytelling to get the message out about our amazing community. We help kubernetes contributors hear the latest news in the most entertaining ways. This can be done through words and visuals. 
```

Comment:
Please capitalize Kubernetes unless it's part of a url; if it's a URL, please link to it too.